common.eai

```diff
@@ -10458,7 +10431,7 @@ function StartUnitAM takes integer ask_qty, integer unitid, integer town, intege
       return UpgradeBuilding(afford_qty, unitid, town, bloc)
     endif
 
-    if unitid == racial_lumber and have_qty < 1 then
+    if unitid == racial_lumber and TownCount(unitid) < 1 then
```

Comment:
Actually should still use have_qty but check town is -1

Reply:
sorry, should use `GetUnitCount(racial_lumber)`